scripts/src/errors.ts

```diff
@@ -291,3 +294,11 @@ export function TooManyRegistrations(message: string): MarketplaceError {
 export function TooMuchEarnOrdered(message: string): MarketplaceError {
 	return TooManyRequests(CODES.TooManyRequests.Amounts, message);
 }
+
+function DeprecationError(index: number, message: string): MarketplaceError {
+	return new MarketplaceError(410, index, "Request to a deprecated resource", message);
+}
+
+export function BlockchainEndpointChanged(message: string): MarketplaceError {
+	return TooManyRequests(CODES.Deprecated.BlockchainEndpointChanged, message);
```

Comment:
TooManyRequests  should be DeprecationError?

Reply:
yep. I haven't added a test yet, I opened the PR just for it to be triggered by travis. But this should be caught by a unit test